warehouse/manage/views/__init__.py

```diff
@@ -1131,6 +1131,10 @@ def __init__(self, project, request):
         self.request = request
         self.project = project
         self.metrics = self.request.find_service(IMetricsService, context=None)
+        self.github_publisher_form = GitHubPublisherForm(
+            self.request.POST,
+            api_token=self.request.registry.settings.get("github.token"),
+        )
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏can you help me understand this change? Before, it was a property, and it's still a property, and doesn't the same thing as before?

Reply:
That's my understanding as well -- I think it's a purely lateral change (it's notionally slightly faster as well, although I don't think that'll be perceivable in this context).

Reply:
I would call these attributes now, not properties. They are functionally the same in usage but this ensures that they are only defined once rather than every time they are accessed.

I started this PR several months ago and I don't remember exactly I made this change but I suspect the properties were causing problems with testing, but also there was no reason for them to be properties in the first place.